openhtf/output/web_gui/__init__.py

```diff
@@ -80,7 +80,6 @@
              default_value=[],
              description='List of manually declared stations.')
 
-
 Hostport = collections.namedtuple('Hostport', ['host', 'port'])
 
 
```

Comment:
As mentioned in person, `TestWatcher.run` is full of race conditions. To be tackled later, but just an FYI to the other maintainers that a read-wait-loop has a race condition between the read and the wait. It should be, in some way, a subscribe-read-wait-loop (where wait returns immediately if a change happened after subscribe, even if it happened before the read). There's a slight chance of increased work, but 0 chance of missing an update.

This will be fixed in a similar manner to how the test binary did it: `threading.Event`